src/app/components/article/Articles.js

```diff
@@ -235,7 +289,7 @@ const ArticlesComponent = ({
                 pageSize={pageSize}
                 onChangePage={handleChangePage}
               />
-              <ExportList filters={adjustFilters(filters)} type={type.replace('-', '_')} />
+              <ExportList filters={adjustFilters(filters)} type={type ? type.replace('-', '_') : 'articles'} />
```

Comment:
Does this mean the backend expects `fact_check`  instead of `fact-check`?

Reply:
Yes: https://github.com/meedan/check-api/blob/develop/lib/list_export.rb#L2